editor/core/files.py

```diff
@@ -84,58 +132,91 @@ def download_file(url: str, file_path: epath.Path):
 def get_dataframe(file_type: FileType, file: io.BytesIO | epath.Path) -> pd.DataFrame:
     """Gets the df associated to the file."""
     if file_type == FileTypes.CSV:
-        return pd.read_csv(file)
+        df = pd.read_csv(file)
     elif file_type == FileTypes.EXCEL:
-        return pd.read_excel(file)
+        df = pd.read_excel(file)
     elif file_type == FileTypes.JSON:
-        return pd.read_json(file)
+        df = pd.read_json(file)
     elif file_type == FileTypes.JSONL:
-        return pd.read_json(file, lines=True)
+        df = pd.read_json(file, lines=True)
     elif file_type == FileTypes.PARQUET:
-        return pd.read_parquet(file)
+        df = pd.read_parquet(file)
     else:
         raise NotImplementedError()
```

Comment:
Can we add a message, something like "File format XXX no supported"? That's less cryptic, and people know what to ask for.

Reply:
Done in another PR (https://github.com/mlcommons/croissant/pull/423/commits/1d2ccf93b23704a522610c68e02a1b165b849298).